src/cdomains/intDomain.ml

```diff
@@ -609,20 +659,35 @@ struct
 
   let cast_to ?torg ?no_ov t = norm ~cast:true t (* norm does all overflow handling *)
 
+  (* This is a container variable for holding the config value of ana.int.interval_threshold_widening *)
```

Comment:
We should not add commented out code to Goblint!

Reply:
Should be fixed by https://github.com/goblint/analyzer/pull/943/commits/8ef8dcf7069a13510af4035ad1d26c64b20b7de9. Thanks for the remark!